configs/config-ccm-psk-tls1_2.h

```diff
@@ -4,8 +4,14 @@
  * \brief Minimal configuration for TLS 1.2 with PSK and AES-CCM ciphersuites
  */
 /*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
- *  SPDX-License-Identifier: Apache-2.0
+ *  Copyright The Mbed TLS Contributors
+ *  SPDX-License-Identifier: Apache-2.0 OR GPL-2.0-or-later
+ *
```

Comment:
Interesting. I wonder why they list both?

Reply:
I don't know, what is even more curious to me is why they did it during a dot release